### ISSUE TITLE
Accessibility improvements

### ISSUE THREAD

**Issue by ChrisTheCat** [tobymao/18xx#1111]
The color options are a good step in the direction of accessibility for people who are color blind. It would be nice if this was extended to all areas of the interface. The colors on the stock market are fixed, so there could be a profile checkbox to enable symbols on the market boxes (following the example of Ticket to Ride, which has a symbol for each color). And someone could check to see if the site is compatible with screen readers. There are probably other improvements that would help people with visual impairments.

I played in-person several times with people who were unable to see the game. It reminded me of my grandfather who could play mental chess. Why not enable that for the site?

It was suggested in Slack that this would be best accomplished in partnership with someone who is an experienced screen reader user.

**Comment by lordscarlet** [comment]
I do work in the federal government space, and we use [ANDI](https://github.com/SSAgov/ANDI) as a first line check for accessibility. Meeting all of its suggestions would probably go a long way to making the site accessible. It certainly won't catch everything (it doesn't seem to really see the map), but its a start.

Additionally, WebAIM is good for color contrast checks. For instance, C&A's card [does not pass](https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=F48221) .

**Comment by lordscarlet** [comment]
This is more usability than accessibility, but they often go hand-in-hand. There are some validations, such as the selling price of privates (e.g. "Price must be between $20 and $80") that should be identified before the user attempts to submit a value.

**Comment by tobymao** [comment]
probably too much to do
